main.tf

```diff
@@ -63,5 +63,7 @@ module "db_instance" {
   backup_retention_period = "${var.backup_retention_period}"
   backup_window           = "${var.backup_window}"
 
+  storage_encrypted = "${var.encrypted}"
```

Comment:
Keep variables names exactly as in Terraform internally. `storage_encrypted` in this case.

Reply:
Also move it after `storage_type`, as these two parameters are related to storage.